irteus/irtrobot.l

```diff
@@ -160,8 +160,11 @@
 		 (send coords :rotate (deg2rad angle) axis wrt) args)))
        (:move-end-pos
 	(let ((coords (send self limb :end-coords :copy-worldcoords))
-	      (pos (pop args)) (wrt (pop args)))
-	  (unless wrt (setq wrt :local))
+	      (pos (pop args)) wrt)
+      (setq wrt
+            (if (or (find (car args) (list :world :local :parent))
```

Comment:
use `memq` https://github.com/euslisp/EusLisp/blob/EusLisp-9.23/lisp/l/coordinates.l#L222 and if that is not `world, local ,parent' how about show warning, please also take care of other 'move-end-*` methods